dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -330,7 +323,9 @@ export const LiveLayout = (props: WebProps | AppsProps) => {
 							showTopBorder={false}
 							showSideBorders={false}
 							padSides={false}
-							backgroundColour={brandBackground.primary}
+							backgroundColour={themePalette(
+								'--article-background',
+							)}
```

Comment:
```suggestion
backgroundColour={sourcePalette.brand[400]}
```

This will fix broken colours. FYI, brandBackground.primary does not equal --article-background